### DISCUSSION TITLE
How can I control my phone in the demo

### DISCUSSION THREAD

**Discussion by Tiny-Fendy** [yume-chan/ya-webadb#416] [Q&A]
I have connect my phone to the demo successfully，but I can't find the way which let me control the phone in web.
<img width="1644" alt="image" src="https://user-images.githubusercontent.com/8954107/167761533-30eacf18-6336-443d-a012-4be9f8b8d215.png">

**Comment by yume-chan** [comment]
You can control your device using the mouse and keyboard. Touchscreen (on computer) support is broken now.

If that doesn't work, please file issues using the template https://github.com/yume-chan/ya-webadb/issues/new?assignees=&labels=bug%2Cdemo&template=demo_bug.yml

don't forget to copy-paste the Scrcpy log in text from here:

![image](https://user-images.githubusercontent.com/1330321/167769570-2e70650c-df85-4951-8789-b0ca41ab62e2.png)

**Reply by Tiny-Fendy** [reply]
I have tried but fail.  Maybe I need to submit a issues.

**Reply by Tiny-Fendy** [reply]
Thanks for your time.
